cookbook/README.md

```diff
@@ -198,9 +198,11 @@ ${COOKBOOKRERUN}_cycle01.log
 ### Final fit cycle
 
 After the user had concluded that the fit has converged to her satisfaction,
-one last run should be made with zero iterations and with `outputStandards =
-True`.  This will ensure the final products have consistently applied superstar
-flats and internal aperture corrections.  This should only take 2-3 minutes.
+one last run should be made with zero iterations (`maxIter=0`). This will
```

Comment:
You are correct that it's confusing that `maxIter=0` is the sentinel that triggers the final cycle output.  Just to be clear, running with 0 iterations doesn't just run quickly, it just means that it will use the atmosphere fit parameters from the previous cycle.
In any event, I have changed this to `maxIterBeforeFinalCycle` (with an improved doc-string) and added a `isFinalCycle` config flag which explicitly says "this is the final cycle and make the appropriate settings".